### PR TITLE
PLT-4891: Added filter to edge_device queryset system_status__is_acti…

### PR DESCRIPTION
**Pull request by tushar-quartic**
Updated the filter queryset for edge_device

### CODE REVIEW

**Review by shivinquartic — DISMISSED** [review]
lgtm

File: notebook/db_util.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Db util class which take care of executing queries.
 
+from deming.edge import device_types
+from deming.models import EnterpriseUser, KernelSession, EdgeDevice
```
Comment (mprasoonquartic) [inline notebook/db_util.py]: curious. this will throw an err to import this before django is setup, right?

Comment (tushar-quartic) [reply notebook/db_util.py]: Formatting the document bought them up ....reverting it

**Review by mprasoonquartic — APPROVED** [review]

**Review by shivinquartic — APPROVED** [review]
lgtm

**Pull request merged**